pkg/globalnet/controllers/node_controller.go

```diff
@@ -52,21 +52,34 @@ func NewNodeController(config *syncer.ResourceSyncerConfig, pool *ipam.IPPool, n
 		nodeName:                   nodeName,
 	}
 
+	clusterCIDR := ""
+	if len(clusterCIDRs) > 0 {
+		clusterCIDR = clusterCIDRs[0]
+	}
+
+	cniIface, err := cni.Discover(clusterCIDR)
```

Comment:
It is not just healthchecker that fails but the host networking connectivity too,  so should we continue here?

Reply:
I'm basically mimicking the behavior from before in that if the CNI IP annotation wasn't set then the controller skipped processing. Here we determine the CNI IP on startup. We don't want to return an error as that would fail the whole startup. We could return and elide the rest of the initialization but the base controller expects a valid `resourceSyncer` instance  to start. We could get around that but I though it's easiest to let the syncer run but skip processing like it did before.